src/Month.js

```diff
@@ -32,7 +32,7 @@ let propTypes = {
   max: PropTypes.instanceOf(Date),
 
   step: PropTypes.number,
-  now: PropTypes.instanceOf(Date),
+  nowAccessor: PropTypes.func.isRequired,
```

Comment:
lets move the default up to Calendar, so it doesn't need to be set a bunch of places (like TimeGrid)

Reply:
Sounds right. I was a bit concerned about backward-compatibility with someone using a naked `<TimeGrid>` etc. but that seems YAGNI-ish.

Reply:
yeah, i appreciate you making the most minimal change, though now is a good time to clean up my messy defaults :P